core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -640,7 +640,8 @@ private[spark] object SparkConf extends Logging {
       DeprecatedConfig("spark.blacklist.killBlacklistedExecutors", "3.1.0",
         "Please use spark.excludeOnFailure.killExcludedExecutors"),
       DeprecatedConfig("spark.yarn.blacklist.executor.launch.blacklisting.enabled", "3.1.0",
-        "Please use spark.yarn.executor.launch.excludeOnFailure.enabled")
+        "Please use spark.yarn.executor.launch.excludeOnFailure.enabled"),
+      DeprecatedConfig("spark.network.remoteReadNioBufferConversion", "3.5.2", "")
```

Comment:
I leave `deprecationMessage` empty, any suggestions?
```
logWarning(
        s"The configuration key '$key' has been deprecated as of Spark ${cfg.version} and " +
        s"may be removed in the future. ${cfg.deprecationMessage}")
```

Reply:
To @pan3793 , actually, that field is supposed to be used to ask the users inform any issues to us.

That's why we cannot say like the following without deprecation.
> while during the whole Spark 3 period, there are no negative reports

Reply:
@dongjoon-hyun how should we fill the message in this case? BTW, according to the comment, I think this is an internal configuration, should we go through the deprecated procedure the same as user-facing configurations too?

> // SPARK-24307 undocumented "escape-hatch" in case there are any issues in converting to
> // ChunkedByteBuffer, to go back to old code-path.  Can be removed post Spark 2.4 if
> // new path is stable.